target/linux/x86/base-files/lib/preinit/01_sysinfo

```diff
@@ -17,7 +17,14 @@ do_sysinfo_x86() {
 
 	for file in product_name board_name; do
 		product="$(cat /sys/devices/virtual/dmi/id/$file 2>/dev/null)"
-		[ -n "$product" ] && break
+		case "$vendor:$product" in
+		*:)
+			break
+			;;
+		"Supermicro:Super Server")
+			break
+			;;
+		esac
```

Comment:
This doesn't seem equivalent at all to me.

Before it was "if $product isn't empty, break". Now it is "if $product *is* empty break". Did you want something like:
```
case "$vendor:$product" in
*: \
"Supermicro:Super Server")
    continue
    ;;
*)
    break
    ;;
esac
```

Reply:
@KanjiMonster Thanks, Jonas. Good eye. Fixed.